modules/hooks.nix

```diff
@@ -14,6 +14,16 @@ in {
               default = [];
             };
         };
+      nix-linter =
+        {
+          checks =
+            mkOption {
+              type = types.listOf types.str;
```

Comment:
I generally prefer `types.attrsOf types.bool` because it lets you both enable and disable, and you can then override individual rules with `lib.mkForce`.

Reply:
Do you propose to have a record with all available extensions as fields an bool flags to enable/disable it?

Reply:
@anpryl I think he meant it to be free form.